kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java

```diff
@@ -110,16 +110,16 @@ public void setPurposeAndMeaningText(PatternDefinition patternDefinition) {
         }
     }
 
-    public void setDescriptionData(PatternField patternFields) {
-
-    }
-
-    public void setFullyQualifiedName(DescrName descrName) {
-        setPropertyValue(FQN_DESCRIPTION_NAME_TEXT, descrName.getNameText());
-    }
-
-    public void setOtherNameText(DescrName descrName) {
-        setPropertyValue(OTHER_NAME_DESCRIPTION_NAME_TEXT, descrName.getNameText());
-    }
+//    public void setDescriptionData(PatternField patternFields) {
```

Comment:
do we want to delete this code instead of commenting it out?

Reply:
agreed.

Reply:
Removed it.